### ISSUE TITLE
Translate to Traditional Chinese (zh-TW)

### ISSUE THREAD

**Issue by thisistz** [huggingface/course#55]
Hi there 👋

Let's translate the course to Traditional Chinese so that the whole community can benefit from this resource 🌎!

Below are the chapters and files that need translating - let us know here if you'd like to translate any and we'll add your name to the list. Once you're finished, open a pull request and tag this issue by including `#issue-number` in the description, where `issue-number` is the number of this issue.

> 🙋 If you'd like others to help you with the translation, you can also post in our [forums](https://discuss.huggingface.co/c/course/20) or tag [@_lewtun](https://twitter.com/_lewtun) on Twitter to gain some visibility.

## Chapters

**0 - Setup**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter0/1.mdx) [@davidpeng86](https://github.com/davidpeng86)

**1 - Transformer models**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/1.mdx) 
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/3.mdx) 
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/4.mdx) 
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/5.mdx) 
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/6.mdx) 
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/7.mdx) 
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/8.mdx) 
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/9.mdx) 
- [ ] [`10.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/10.mdx)

**2 - Using 🤗 Transformers**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/1.mdx) [snorlaxchoi](https://github.com/snorlaxchoi)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/2.mdx) [snorlaxchoi](https://github.com/snorlaxchoi)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/3.mdx) [snorlaxchoi](https://github.com/snorlaxchoi)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/8.mdx)

**3 - Fine-tuning a pretrained model**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3.mdx)
- [ ] [`3_tf.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3_tf.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/6.mdx)

**4 - Sharing models and tokenizers**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/6.mdx)

**5 - The 🤗 Datasets library**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/8.mdx)

**6 -  The 🤗 Tokenizers library**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/3.mdx)
- [ ] [`3b.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/3b.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/8.mdx)
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/9.mdx)
- [ ] [`10.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/10.mdx)

**7 - Main NLP tasks**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/8.mdx)
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/9.mdx)

**8 - How to ask for help**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/4.mdx)
- [ ] [`4_tf.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/4_tf.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/7.mdx)

**Events**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/event/1.mdx)

**Comment by 1375626371** [comment]
Hello, we are college students studying natural language processing, last month we started the translation of huggingface's course. There was no translation project at the time, so we scraped web pages using crawler, translated them using machine translation, and did the first proofreading by hand. Next, we will carry out the second proofreading and establish a comparison table of professional vocabulary. We want to contribute parts of our translation, can we participate together?

**Comment by davidpeng86** [comment]
Absolutely. Post the chapters you'd like to work on so we don't end up overlapping on chapters. I just started and am halfway through the setup so there is much to work on.

**Comment by lewtun** [comment]
Wow @1375626371 amazing to hear that you already started working on a translation 🤯 ! As @davidpeng86 says, you're more than welcome to contribute - just indicate here which chapters you'd like to start with and we can add your name to the list. Thank you!

**Comment by 1375626371** [comment]
It's great that we can contribute to the open source community while learning. in the next week. We will convert Chapters1 while trying to build our translation process. @petrichor1122@zhlhyx are two of my great teammates who will give their respective translation plans for the coming week.
I plan on translating sections 5, 6, 8, 10 in chapter 1 for the next week

**Comment by lewtun** [comment]
Thank you very much @1375626371 and @zhlhyx - I've marked both your names on the sections. Looking forward to your PRs!

**Comment by oToToT** [comment]
Hi, will this translation be in Traditional Chinese or Simplified Chinese?

As I know, either ISO 639-1 or ISO 639-2 makes no differences between them. However, they actually differ in many aspects including words, characters.  
Therefore, many websites have different pages for them. For example, MDN [(zh-TW)](https://developer.mozilla.org/zh-TW/docs/Web)/[(zh-CN)](https://developer.mozilla.org/zh-CN/docs/Web), Wikipedia [(zh-tw)](https://zh.wikipedia.org/zh-tw/Wikipedia:首頁)/[(zh-cn)](https://zh.wikipedia.org/zh-cn/Wikipedia:首頁), and so on.

Should we also have different pages for them?

**Comment by John-China** [comment]
Agree. [This table](https://hackmd.io/@l10n-tw/glossaries) alone shows a ton of difference between Traditional Chinese and Simplified Chinese.

**Comment by davidpeng86** [comment]
I will be translating into Traditional Chinese.

**Comment by lewtun** [comment]
> Hi, will this translation be in Traditional Chinese or Simplified Chinese?

This is a great question @oToToT! You're more than welcome to have different pages for traditional and simplified Chinese - the only thing to note is that you'll need to have separate folders for the content (e.g. `zh-CN` and `zh-TW`). 

If you wish to add simplified Chinese, I suggest opening a new issue with something like "Translate to Simplified Chinese" to distinguish it from this issue & make it easier to see which sections have already been translated.

**Comment by petrichor1122** [comment]
I'm very glad to participate in this project. I plan to translate sections 3 and 9 in chapter 1 next week. Please mark it for me. Thank you!

**Comment by lewtun** [comment]
Thank you @petrichor1122 - I've added your name to the list! This is really exciting as we'll soon have the first chapter available in traditional Chinese 🥳 !

**Comment by snorlaxchoi** [comment]
Hi everyone ☺️, I'm a freelance translator and currently working on a NLP side-project and I absolutely love huggingface for helping me out a lot in my project so I would love to contribute to the community. Since @petrichor1122 will be translating sections 3 and 9 of chapter 1, I plan to translate sections 2,3,4 of chapter 1 in Traditional Chinese. Please mark it for me. Thanks!

**Comment by 1375626371** [comment]
The translation work for the last week has been basically completed. I plan to translate sections 8 in chapter 2 next week. Please mark it for me. Thank you!

**Comment by snorlaxchoi** [comment]
> Thank you @petrichor1122 - I've added your name to the list! This is really exciting as we'll soon have the first chapter available in traditional Chinese 🥳 !

I have gone through the first chapter and it seems to be in simplified Chinese, so taken @oToToT's idea, I'll probably have to move to another thread for Traditional Chinese so that it wouldn't clash with this project. :)

**Comment by davidpeng86** [comment]
> I have gone through the first chapter and it seems to be in simplified Chinese, so taken @oToToT's idea, I'll probably have to move to another thread for Traditional Chinese so that it wouldn't clash with this project. :)

I made this thread with the intent of translating into traditional Chinese. It would be great if @lewtun can change the [zh](https://github.com/huggingface/course/tree/main/chapters/zh) file name into zh-CN.

**Comment by lewtun** [comment]
Thanks for the clarification @snorlaxchoi and @davidpeng86! I'll update the language tags accordingly & mark the completed chapter in the new issue

**Comment by lewtun** [comment]
OK everyone, I've created a new issue for those who wish to continue work on the Simplified Chinese translation: https://github.com/huggingface/course/issues/118

Let's keep the current issue for Traditional Chinese as @davidpeng86 mentioned - this means we can close https://github.com/huggingface/course/issues/117 right @snorlaxchoi ?

Also @1375626371 @zhlhyx @petrichor1122 - I plan to rename the folder structure to be `zh-CN` for your contributions in Chapter 1. 

Hope this isn't too confusing!

**Comment by 1375626371** [comment]
> OK everyone, I've created a new issue for those who wish to continue work on the Simplified Chinese translation: #118
> 
> Let's keep the current issue for Traditional Chinese as @davidpeng86 mentioned - this means we can close #117 right @snorlaxchoi ?
> 
> Also @1375626371 @zhlhyx @petrichor1122 - I plan to rename the folder structure to be `zh-CN` for your contributions in Chapter 1.
> 
> Hope this isn't too confusing!

Great, the grouping is correct now. Our translation is indeed simplified Chinese, thank you. We will submit our translations in Simplified Chinese (zh-CN) later.

**Comment by snorlaxchoi** [comment]
> OK everyone, I've created a new issue for those who wish to continue work on the Simplified Chinese translation: #118
> 
> Let's keep the current issue for Traditional Chinese as @davidpeng86 mentioned - this means we can close #117 right @snorlaxchoi ?
> 
> Also @1375626371 @zhlhyx @petrichor1122 - I plan to rename the folder structure to be `zh-CN` for your contributions in Chapter 1.
> 
> Hope this isn't too confusing!

Yes, thank you, please close #117 and I will continue to work on translating into Traditional Chinese under this thread.

**Comment by ateliershen** [comment]
Hi, I noticed this thread has been inactive for 7 months already.
I am keen to pick up this task, can I still work on it? Thanks.

**Comment by thisistz** [comment]
@ateliershen yeah feel free to hop on

**Comment by thliang01** [comment]
Hey @thisistz - Following the recent updates to the English version of the course materials, would it be appropriate to submit a translation request for Chapter 3?

**Comment by thliang01** [comment]
Hi everyone! 👋

I'm [@thliang01](https://github.com/thliang01) and I'd like to claim **Chapter 2 (使用 🤗 Transformers)** for the zh-TW translation.

I noticed the existing zh-TW content in [PR #477](https://github.com/huggingface/course/pull/477) was converted from Simplified Chinese (zh-CN) via OpenCC. I'd like to redo Chapter 2 by **translating directly from the English source**, since OpenCC does not produce natural Traditional Chinese — vocabulary and phrasing used in Taiwan differ in ways mechanical conversion cannot fix (e.g. `加载` → `載入`, `数据` → `資料`, `代码` → `程式碼`).

My plan for Chapter 2:
- Rewrite sections 1–7 from English (fixing OpenCC-converted content)
- Translate the new section 8 — *Optimized Inference Deployment* (added to English after PR #477 merged)
- Move the quiz to section 9 to match the updated English structure
- Update `_toctree.yml` to reflect the new section titles

I'll submit as small, focused PRs as suggested by @lewtun. May I proceed with Chapter 2?

**Comment by thisistz** [comment]
> Hi everyone! 👋
> 
> I'm [@thliang01](https://github.com/thliang01) and I'd like to claim **Chapter 2 (使用 🤗 Transformers)** for the zh-TW translation.
> 
> I noticed the existing zh-TW content in [PR #477](https://github.com/huggingface/course/pull/477) was converted from Simplified Chinese (zh-CN) via OpenCC. I'd like to redo Chapter 2 by **translating directly from the English source**, since OpenCC does not produce natural Traditional Chinese — vocabulary and phrasing used in Taiwan differ in ways mechanical conversion cannot fix (e.g. `加载` → `載入`, `数据` → `資料`, `代码` → `程式碼`).
> 
> My plan for Chapter 2:
> - Rewrite sections 1–7 from English (fixing OpenCC-converted content)
> - Translate the new section 8 — *Optimized Inference Deployment* (added to English after PR #477 merged)
> - Move the quiz to section 9 to match the updated English structure
> - Update `_toctree.yml` to reflect the new section titles
> 
> I'll submit as small, focused PRs as suggested by @lewtun. May I proceed with Chapter 2?

Yes please do, we're glad to have your input
